python/sparkdl/transformers/tf_image.py

```diff
@@ -215,15 +218,11 @@ def _convertOutputToImage(self, df, tfs_output_col, output_shape):
         width = int(output_shape[2])
         def to_image(orig_image, numeric_data):
             # Assume the returned image has float pixels but same #channels as input
-            mode = orig_image.mode if orig_image.mode == "float32" else "RGB-float32"
-            return [mode, height, width, orig_image.nChannels,
-                    bytearray(np.array(numeric_data).astype(np.float32).tobytes())]
-        to_image_udf = udf(to_image, imageSchema)
-        return (
-            df.withColumn(self.getOutputCol(),
-                          to_image_udf(df[self.getInputCol()], df[tfs_output_col]))
-              .drop(tfs_output_col)
-        )
+            mode = imageIO.imageTypeByName('CV_32FC%d' % orig_image.nChannels)
+            return Row(origin="",mode=mode.ord, height=height, width=width, nChannels=orig_image.nChannels, data=bytearray(np.array(numeric_data).astype(np.float32).tobytes()))
```

Comment:
Nit (style): Try to keep line length < 100 chars if possible

Reply:
We have a Scala style guide (https://github.com/databricks/scala-style-guide) but I can't find one for Python -- AFAIK for Python we just follow PEP8 conventions.

Reply:
kk, makes sense :) I'll try to follow that in the future.